packages/blockchain-wallet-v4-frontend/src/modals/CoinifyExchangeData/Payment/template.success.js

```diff
@@ -178,34 +187,22 @@ const Payment = props => {
       <PaymentColRight>
         <PaymentColRightInner>
           <ButtonContainer>
-            {path(['name'], level) < 2 && medium === 'bank' ? (
-              <Button
-                nature='primary'
-                fullwidth
-                onClick={triggerKyc}
-                disabled={!medium || busy}
-              >
-                {busyHelper(busy)}
-              </Button>
-            ) : (
-              <StepTransition
-                next
-                Component={Button}
-                nature='primary'
-                fullwidth
-                disabled={!medium || busy}
-              >
-                {busyHelper(busy)}
-              </StepTransition>
-            )}
+            <Button
+              nature='primary'
+              fullwidth
+              onClick={() => onClickHelper()}
```

Comment:
`() =>` usage of callback props recreates them on each render, it's better to use `onClick={onClickHelper}` if it's not becoming too bulky(e.g. binds)